setup.py

```diff
@@ -33,7 +33,7 @@ def read_version():
 
 # Declare minimal set for installation
 required_packages = [
-    "attrs==20.3.0",
+    "attrs==21.4.0",
```

Comment:
```suggestion
    "attrs>=21.4.0",
```
can you add a little flexibility here?